lib/pages/chat/chat_app_bar_list_tile.dart

```diff
@@ -33,7 +34,7 @@ class ChatAppBarListTile extends StatelessWidget {
             child: Padding(
               padding: const EdgeInsets.symmetric(horizontal: 4.0),
               child: Linkify(
-                text: title,
+                text: title.bidiFormatted,
```

Comment:
Now every time I want to add a text somewhere, I need to think about adding the `.bidiFormatted` suffix? I would prefer a solution, which does not add more complexity